docs/modules/ROOT/pages/accounts.adoc

```diff
@@ -98,3 +98,196 @@ But if you don't have an account to invoke it, you will probably want to use the
 
 To do counterfactual deployments, you need to implement another protocol-level entrypoint named
 `\\__validate_deploy__`. Check the {counterfactual} guide to learn how.
+
+== Usage
+
+Contracts for Cairo offers two flavors of components to use for creating accounts: `AccountComponent` and `EthAccountComponent`.
+Though the ISRC6 interface is agnostic regarding validation schemes, both available components utilize public-private key pairs that control the account.
+The public key is set in the constructor.
+
+Mixin implementations are available to use for both components which include basic account functionality as well as additional features such as the ability to:
+
+- View and transfer the account's public key.
+- Declare contract class hashes.
+- Counterfactually deploy.
+
+=== AccountComponent
+
+:account-mixin: xref:/api/account.adoc#AccountComponent-Embeddable-Mixin-Impl[AccountMixin]
+:initializer: xref:/api/account.adoc#AccountComponent-initializer[initializer]
+:custom-account-setup: https://foundry-rs.github.io/starknet-foundry/starknet/account.html#custom-account-contract[custom account setup]
+
+To create an account, implement the {account-mixin} and set up the constructor to initialize the contract.
+The {initializer} method sets the public key for the account contract and registers the account interface id.
+A vanilla account contract looks like this:
+
+```[,cairo]
+#[starknet::contract(account)]
+mod MyAccount {
+    use openzeppelin::account::AccountComponent;
+    use openzeppelin::introspection::src5::SRC5Component;
+
+    component!(path: AccountComponent, storage: account, event: AccountEvent);
+    component!(path: SRC5Component, storage: src5, event: SRC5Event);
+
+    // Account Mixin
+    #[abi(embed_v0)]
+    impl AccountMixinImpl = AccountComponent::AccountMixinImpl<ContractState>;
+    impl AccountInternalImpl = AccountComponent::InternalImpl<ContractState>;
+
+    #[abi(embed_v0)]
+    impl SRC5Impl = SRC5Component::SRC5Impl<ContractState>;
+
+    #[storage]
+    struct Storage {
+        #[substorage(v0)]
+        account: AccountComponent::Storage,
+        #[substorage(v0)]
+        src5: SRC5Component::Storage
+    }
+
+    #[event]
+    #[derive(Drop, starknet::Event)]
+    enum Event {
+        #[flat]
+        AccountEvent: AccountComponent::Event,
+        #[flat]
+        SRC5Event: SRC5Component::Event
+    }
+
+    #[constructor]
+    fn constructor(ref self: ContractState, public_key: felt252) {
+        self.account.initializer(public_key);
+    }
+}
+```
+
+To deploy the Contracts for Cairo account variant, compile the contract and declare the class hash because custom accounts are likely not declared on the network.
```

Comment:
```suggestion
To deploy the Contracts for Cairo account variant, compile the contract and declare the class hash because custom accounts are likely not declared.
```